Explorer/Assets/Scripts/ECS/Unity/Textures/Components/VideoTextureConsumer.cs

```diff
@@ -1,11 +1,53 @@
 using ECS.StreamableLoading.Textures;
 using System;
+using System.Collections.Generic;
 using UnityEngine;
 
 namespace ECS.Unity.Textures.Components
 {
     public struct VideoTextureConsumer : IDisposable
     {
+        /// <summary>
+        /// Gets the current world position of the maximum corner of the bounding box that contains all the mesh renderers used by video consumers of one texture.
+        /// </summary>
+        public Vector3 BoundsMax
```

Comment:
You don't need to calculate bounds every frame: you can do it only once when the list of renderers is modified

Reply:
I need to know the world position of each corner. When renderers are added, there is a real risk that their position is wrong.